test/sanity-check/sanity.js

```diff
@@ -20,7 +21,6 @@ require('./api/globalfield-test')
 require('./api/release-test')
 require('./api/label-test')
 require('./api/contentType-delete-test')
-require('./api/taxonomy-test')
 require('./api/terms-test')
```

Comment:
@harshithad0703 ,Since taxonomy and terms are global modules and won't throw an error if executed after a stack test, I believe you can keep them together after the stack test.

Reply:
yes, will have the test suites together